sdk/tables/data-tables/samples-dev/generateTableSAS.ts

```diff
@@ -82,7 +82,7 @@ async function generateTableSasSample() {
 
   // Create the table SAS token
   const tableSas = generateTableSas(tableName, cred, {
-    expiresOn: new Date("2021-12-12"),
+    expiresOn: new Date(Date.now() + 60 * 60 * 1000),
```

Comment:
It would help readability if you document what `Date.now() + 60 * 60 * 1000` means (An hour from now). I'd recommend either putting it into a variable.

```typescript
const anHourFromNow = Date.now() + 60 * 60 * 1000;
const tableSas = generateTableSas(tableName, cred, {
  expiresOn: new Date(anHourFromNow ),
  permissions: tablePermissions,
});
```

OR

```typescript
const tableSas = generateTableSas(tableName, cred, {
  /** Setting expiration date to be an hour from the time this line executes */
  expiresOn: new Date(Date.now() + 60 * 60 * 1000),
  permissions: tablePermissions,
});
```